src/clientiface.cpp

```diff
@@ -384,6 +385,8 @@ void RemoteClient::GetNextBlocks (
 		if (d > full_d_max) {
 			new_nearest_unsent_d = 0;
 			m_nothing_to_send_pause_timer = 2.0f;
+			infostream << "Server: Time to send complete map: " << m_map_send_completion_timer << "s" << std::endl;
+			m_map_send_completion_timer = 0.0f;
```

Comment:
this needs to be per-client somehow
also maybe this better fits into the profiler?

Reply:
Hmm... Good point. EIther log the client or send to the profiler.

Reply:
I think profiler is actually not a good idea, as this value would not be available frequently. I'll log the player name instead... It's really just an aid for a server admin to set the unload timeout correctly - or for us DEVs to see what is going on.

Reply:
We also do not log the client for other map related changes, so that might be confusing.